service/src/config.rs

```diff
@@ -58,6 +59,7 @@ impl From<MainConfig> for Config {
                 serve_auth_token: value.service.serve_auth_token,
                 deployment: value.subgraphs.escrow.config.deployment_id,
                 query_url: value.subgraphs.escrow.config.query_url.into(),
+                query_auth_token: value.subgraphs.network.config.query_auth_token.clone(),
```

Comment:
```suggestion
                query_auth_token: value.subgraphs.network.config.query_auth_token,
```
The second doesn't need to be cloned.